### PR TITLE
add article on deploying spark with oshinko cli

### PR DESCRIPTION
**Pull request by elmiko**
This change adds the new how do i article for deploying a cluster with
oshinko-cli. It also updates links and the title of the oshinko-cli
install how do i article.

### CODE REVIEW

File: _howdoi/deploy-a-spark-cluster-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+* A terminal shell and OpenShift `oc` tool available with an active login to
+  OpenShift.
+
+* An OpenShift project with the radanalytics.io manifest installed, see the
+  link:/get-started[Get Started] instructions for more help.
+
+.Procedure
+
+. Type `oshinko create <cluster name>` into your terminal shell, replacing
+  `<cluster name>` with the name you will use to identify the cluster.
```
Comment (rebeccaSimmonds19) [inline _howdoi/deploy-a-spark-cluster-cli.adoc]: When the cluster is created it now also tells you the spark master URL, you could add this in if you wanted. https://github.com/radanalyticsio/oshinko-cli/pull/127 - PR that solved this

Comment (rebeccaSimmonds19) [reply _howdoi/deploy-a-spark-cluster-cli.adoc]: Otherwise lgtm

Comment (elmiko) [reply _howdoi/deploy-a-spark-cluster-cli.adoc]: i will add an example, my only concern here is that our tooling could change and i don't want the docs to get stale that fast.

**Comment by elmiko** [comment]
@rebeccaSimmonds19 added some code examples, ptal

**Pull request merged**